suite/tests/client-interface/attach_test.template

```diff
@@ -4,8 +4,8 @@ starting attachee
 starting
 #endif
 thank you for testing attach
-thread init
 event_post_attach
+thread init
```

Comment:
Oh ok you're saying the Linux ones fail with the new order.  You can use `#ifdef WINDOWS` (just like below on line 9) to make the two different orders.

Reply:
fix it.